### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arkadykanevsky**
Move readme message to latest relesed version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ JetPack code and documentation is distributed under the Apache 2 license. Contr
 
 JetPack is an innovative automation package that is used to configure the infrastructure hardware and OpenStack software in a fully automated fashion. The toolkit includes scripts from Dell EMC that work with Red Hat OpenStack Platform Director to automate the deployment process, not only to save time but to ensure the process is reliable and repeatable each time.
 
-Please reference the current Dell EMC NFV Ready Bundle for Red Hat Software Deployment Guide v10.2 for installation and configuration at:
-https://www.dell.com/support/article/us/en/19/sln310368/dell-emc-nfv-ready-bundle-for-red-hat
+Please reference the current Dell EMC NFV Ready Bundle for Red Hat Software Deployment Guide v13.0 for installation and configuration at:
```
Comment (heydog15) [inline README.md]: Should we still call this a Ready Bundle?

File: README.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ JetPack code and documentation is distributed under the Apache 2 license. Contr
...
+https://github.com/dsp-jetpack/JetPack/blob/master/doc/Dell_EMC_Red_Hat_Ready_Architecture_Deployment_Guide_Notes_v13.0.docx
```
Comment (heydog15) [inline README.md]: Don't understand why the word Notes is in the title of the Guide?  Should be just Dell_EMC_Red_Hat_Ready_Architecture_Deployment_Guide_v13.0.docx

Comment (dpaterson) [reply README.md]: The link is correct and file can be saved, so the file will have to be renamed and re-committed if "Notes" shouldn't be there.

Comment (gaelrehault) [reply README.md]: Can you point to new doc format/location : https://github.com/dsp-jetpack/JetPack/blob/master/doc/deployment_guide.md

**Comment by arkadykanevsky** [comment]
updated doc pointer

**Comment by arkadykanevsky** [comment]
Not sure why I see failed check.

**Comment by gaelrehault** [comment]
Could you update the solution name as per Kurt s comment ?

**Pull request merged**